ci/Jenkinsfile.combined

```diff
@@ -61,13 +61,11 @@ pipeline {
           linux_e2e = jenkins.Build('status-desktop/systems/linux/x86_64/tests-e2e-old')
         } } }
         stage('Linux/E2E/new') { steps { script {
-          def Date date = new Date()
-          def String datePart = date.format("dd.MM.yyyy")
           linux_e2e = build(
             job: 'status-desktop/systems/linux/x86_64/tests-e2e-new',
             parameters: jenkins.mapToParams([
               BUILD_SOURCE:       linux_x86_64.fullProjectName, 
-              TESTRAIL_RUN_NAME: "Nightly regression ${datePart}"  
+              TESTRAIL_RUN_NAME: "Nightly regression"  
```

Comment:
That title doesn't really make sense if you run this manually or as a release tho.